### PR TITLE
2024-06-26 meeting minutes

### PR DESCRIPTION
**Pull request by santosomar**
Meeting minutes for the CSAF TC monthly call hosted on 2024-06-26.

### CODE REVIEW

File: meeting_minutes/2024/2024-06-26.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+## Roll call
+
+Inability to register attendees due to OASIS system challenges.
+
+## Participants
+
+| Given Name | Family Name | Affiliation                                                 | Role                        |
+|:-----------|:------------|:------------------------------------------------------------|:----------------------------|
+| Stefan     | Hagen       | Individual                                                  | Voting Member, taking notes |
```
Comment (sthagen) [inline meeting_minutes/2024/2024-06-26.md:20]: Did I really take notes? Not so sure ...

**Review by sthagen — APPROVED** [review]
I guess the note taker was not me but besides this I do not object merge.

File: meeting_minutes/2024/2024-06-26.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+- Pull-request Add list of organizations publishing CSAF on website #686.
+  - Use organizations out there to spread and encouraging adoptions.  Who’s using the standard – use names on website for recognition and awareness. 
+  - Simple as a list or open to suggestions on how to provide that information.
+  - Covered by CSAF aggregators? Is that their purpose.  They collect who they are publishing, CSAF may not be compliant, but can’t retrieve automatically.  
+  - Value to have a list but who would collect and maintain.  Include in CSAF 2.1 field to announce it.
+  - Can collect names and find way to convert it over. Promote data quality throughout system. 
+  - Discuss more at next meeting. 
+- Miscellaneous comments:
+  - Sonny van Lingen (Huawei) 6/26/2024 1:18 PM • Thanks for this recap :-)
+  - Thomas Schmidt 6/26/2024 1:25 PM • oasis-csaf@connectedcommunity.org
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-06-26.md:107]:
```suggestion

File: meeting_minutes/2024/2024-06-26.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  - Vivek Nair (MSFT) 6/26/2024 1:52 PM • Have we looked at the new CNA rules (https://www.cve.org/Media/News/item/blog/2024/05/07/CNA-Rules-v4-0-Update-and-Transition) and its impact on CSAF? 
+
+    I'll raise it on GitHub for discussion next time - If CVE's are to be assigned for Cloud/AI/ML regardless of the need for action by the customer, we might need to distinguish between CVE's requiring action vs those which do not (like, maybe by a flag?).
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-06-26.md:110]:
```suggestion
  - Vivek Nair: Have we looked at the [new CNA rules](https://www.cve.org/Media/News/item/blog/2024/05/07/CNA-Rules-v4-0-Update-and-Transition) and its impact on CSAF?
    - will raise it on GitHub for discussion next time - If CVE's are to be assigned for Cloud/AI/ML regardless of the need for action by the customer, we might need to distinguish between CVE's requiring action vs those which do not (like, maybe by a flag?).

File: meeting_minutes/2024/2024-06-26.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+- Pull-reqeust Add rule that new verisons SHOULD oly be released on significant changes #751
+  - Martin Prpic – scalability of revision history. 
+  - Guidance of what should contain and guidelines and automated history.  
+  - No comments other than that.
+  - Will work with Martin offline Thomas Schmidt to make it more feasible. 
+- Pull-request Add list of organizations publishing CSAF on website #686.
+  - Use organizations out there to spread and encouraging adoptions.  Who’s using the standard – use names on website for recognition and awareness. 
+  - Simple as a list or open to suggestions on how to provide that information.
+  - Covered by CSAF aggregators? Is that their purpose.  They collect who they are publishing, CSAF may not be compliant, but can’t retrieve automatically.
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-06-26.md:101]:
```suggestion
  - Covered by CSAF aggregators? Is that their purpose. They decide who they are collecting. CSAF publishers may be compliant, but can’t retrieve automatically (and therefore hard to find).

File: meeting_minutes/2024/2024-06-26.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  - Guidance of what should contain and guidelines and automated history.  
+  - No comments other than that.
+  - Will work with Martin offline Thomas Schmidt to make it more feasible. 
+- Pull-request Add list of organizations publishing CSAF on website #686.
+  - Use organizations out there to spread and encouraging adoptions.  Who’s using the standard – use names on website for recognition and awareness. 
+  - Simple as a list or open to suggestions on how to provide that information.
+  - Covered by CSAF aggregators? Is that their purpose.  They collect who they are publishing, CSAF may not be compliant, but can’t retrieve automatically.  
+  - Value to have a list but who would collect and maintain.  Include in CSAF 2.1 field to announce it.
+  - Can collect names and find way to convert it over. Promote data quality throughout system. 
+  - Discuss more at next meeting.
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-06-26.md:104]:
```suggestion
 - Sonny van Lingen: And at least theoretically one could run a CSAF Trusted Provider with `"list_on_csaf_aggregators": "false"` and `"mirror_on_csaf_aggregators": "false"` :-)
 - Value to have a list but who would collect and maintain. Include in CSAF 2.1 field to announce it.
  - Martin Prpic (Red Hat): Could just start collecting vendor names in that issue and then convert it into a list for the website.
  - Can collect names and find way to convert it over. Promote data quality throughout system.
  - Thomas Schmidt: starting points could be [CSAF lister](https://wid.cert-bund.de/.well-known/csaf-aggregator/aggregator.json) and [CSAF aggregator](https://aggregator.cert-bund.de/.well-known/csaf-aggregator/aggregator.json) of BSI
  - Discuss more at next meeting.

File: meeting_minutes/2024/2024-06-26.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+- Miscellaneous comments:
+  - Sonny van Lingen (Huawei) 6/26/2024 1:18 PM • Thanks for this recap :-)
+  - Thomas Schmidt 6/26/2024 1:25 PM • oasis-csaf@connectedcommunity.org
+  - Vivek Nair (MSFT) 6/26/2024 1:52 PM • Have we looked at the new CNA rules (https://www.cve.org/Media/News/item/blog/2024/05/07/CNA-Rules-v4-0-Update-and-Transition) and its impact on CSAF? 
+
+    I'll raise it on GitHub for discussion next time - If CVE's are to be assigned for Cloud/AI/ML regardless of the need for action by the customer, we might need to distinguish between CVE's requiring action vs those which do not (like, maybe by a flag?).
+  - Sonny van Lingen (Huawei) 6/26/2024 1:58 PM • And at least theoretically one could run a CSAF Trusted Provider with list_on_csaf_aggregators "false" and mirror_on_csaf_aggregators "false" :-)
+  - Martin Prpic (Red Hat) 6/26/2024 1:58 PM • Could just start collecting vendor names in that issue and then convert it into a list for the website.
+  - Thomas Schmidt 6/26/2024 1:59 PM • CSAF lister: wid.cert-bund.de/.well-known/csaf-aggregator/aggregator.json
+  - CSAF aggregator: aggregator.cert-bund.de/.well-known/csaf-aggregator/aggregator.json
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-06-26.md:114]: They belong to the topic #686 above.
```suggestion

File: meeting_minutes/2024/2024-06-26.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+- The issue of allowing sharing groups in the distribution section of CSAF advisories was raised, with the aim of better identifying who receives the advisories.
+- The meeting discussed issues with the scalability of adding individual revision history entries for Red Hat CSAF documents.
+- There was a suggestion to provide guidance in the standard for what the revision history should contain and how to implement automated revision histories.
+- The idea of creating a list of organizations that have adopted the CSAF standard was proposed, with potential methods for collecting and displaying this information discussed.
+
+
+
+## Adjourn
+
+- The meeting was adjourned @ 18:058 UTC
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-06-26.md:128]:
```suggestion
- The meeting was adjourned @ 17:58 UTC

File: meeting_minutes/2024/2024-06-26.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+
+**Note**: All monthly meetings take place on the last Wednesday of each month at 18:00 UTC (19:00 CET, 13:00 EST, 10:00 PST).
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-06-26.md:130]: As the time is always the same (except for the European/US daylight saving shift weeks), I think we are meeting according to ET...
```suggestion
**Note**: All monthly meetings take place on the last Wednesday of each month at 13:00 ET (19:00 CEST/19:00 CET, 10:00 PT, 17:00/18:00 UTC).

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
Thank you for taking note. Some comments below

**Comment by santosomar** [comment]
I believe that I captured all of your suggestions @tschmidtb51  in the current commits. Please check.

File: meeting_minutes/2024/2024-06-26.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+  - Would like TC to approve. 
+  - Not a request to make EPSS mandatory or only scoring method, option to also put EPSS into to deliver into CSAF if want to constituency. 
+  - Optional fields can be omitted by the author
+  - Put issues in comments and revisit in next meeting and look into offline.
+- Issue – Adding customer specific publisher sub-item #705
+  - After meeting Thomas Schmidt will send email to put forth motion to approve.
+- Issue – Suggestion: add a type to a link #722.
+  - It’s very broad they would like to sub-categories on references.  
+  - Put more to machine readable directions. 
+  - Call for action via email Thomas Schimdt to send out.
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-06-26.md:91]:
```suggestion
- Call for action via email Thomas Schmidt to send out.

File: meeting_minutes/2024/2024-06-26.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+- Issue – #662 Add remediation category “fix_planned”!
+  - Michael Reeder wanted to bring it up again.  
+  - Left off for further discussion about feasibility.  
+  - Tobi not in meeting today to weigh in.
+- Issue – #665 Vulnerabilities Property – remediations – Category
+  - Thomas Proell closed issue 665 as not working in CSAF 2.1 
+  - In that regard, assume will look into later version to CSAF 3.0 (Thomas Schmidt)
+  - As 665 blocking off table can proceed with 662.
+  - Michael to send a motion to Accept 662 for CSAF 2.1 to TC via [email list](oasis-csaf@connectedcommunity.org)
+- Issue – Enforce product_version_range rules #636 (presented by Thomas Schmidt)
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-06-26.md:74]:
```suggestion
- Issue – Enforce `product_version_range` rules #636 (presented by Thomas Schmidt)

File: meeting_minutes/2024/2024-06-26.md [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+- Issue – Suggestion: add a type to a link #722.
+  - It’s very broad they would like to sub-categories on references.  
+  - Put more to machine readable directions. 
+  - Call for action via email Thomas Schimdt to send out. 
+
+- Issue – Add rule that new versions SHOULD only be released on significant changes #751
+  - Martin Prpic – scalability of revision history. 
+  - Guidance of what should contain and guidelines and automated history.  
+  - No comments other than that.
+  - Will work with Martin offline Thomas Schmidt to make it more feasible.
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-06-26.md:97]:
```suggestion
  - Thomas Schmidt will work with Martin offline to make it more feasible.

File: meeting_minutes/2024/2024-06-26.md [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+  - Martin Prpic – scalability of revision history. 
+  - Guidance of what should contain and guidelines and automated history.  
+  - No comments other than that.
+  - Will work with Martin offline Thomas Schmidt to make it more feasible. 
+- Issue – Add list of organizations publishing CSAF on website #686.
+  - Use organizations out there to spread and encouraging adoptions.  Who’s using the standard – use names on website for recognition and awareness. 
+  - Simple as a list or open to suggestions on how to provide that information.
+  - Covered by CSAF aggregators? Is that their purpose. They decide who they are collecting. CSAF publishers may be compliant, but can’t retrieve automatically (and therefore hard to find).
+ - Sonny van Lingen: And at least theoretically one could run a CSAF Trusted Provider with `"list_on_csaf_aggregators": "false"` and `"mirror_on_csaf_aggregators": "false"` :-)
+ - Value to have a list but who would collect and maintain. Include in CSAF 2.1 field to announce it.
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-06-26.md:103]:
```suggestion
  - Sonny van Lingen: And at least theoretically one could run a CSAF Trusted Provider with `"list_on_csaf_aggregators": "false"` and `"mirror_on_csaf_aggregators": "false"` :-)
  - Value to have a list but who would collect and maintain. Include in CSAF 2.1 field to announce it.

**Review by tschmidtb51 — COMMENTED** [review]
@santosomar Sorry - looks like I missed some things in my first review

**Comment by sthagen** [comment]
Let us not make these minutes too beautiful 🙈 ... LGTM, still.

**Review by tschmidtb51 — APPROVED** [review]
LGTM

**Comment by santosomar** [comment]
Motion for approval at: https://groups.oasis-open.org/discussion/motion-to-approve-csaf-june-2024-meeting-minutes

**Comment by tschmidtb51** [comment]
Motion moved and seconded: https://groups.oasis-open.org/discussion/motion-to-approve-csaf-june-2024-meeting-minutes

**Pull request merged**